src/pcapi/scripts/setup_app_search.py

```diff
@@ -3,77 +3,71 @@
 import sys
 
 from pcapi.core.search.backends import appsearch
+from pcapi.core.search.backends.appsearch import OFFERS_ENGINE_NAME
+from pcapi.core.search.backends.appsearch import VENUES_ENGINE_NAME
 from pcapi.flask_app import app
 
 
-def _check_errors(response):
-    if not response.ok:
-        result = response.json()
-        try:
-            errors = [result["error"]]
-        except KeyError:
-            errors = result["errors"]
-        print("\n".join(f"ERR: {error}" for error in errors))
-        return False
-    return True
-
-
-def setup_offers_engine():
-    offers_engine = appsearch.AppSearchBackend().offers_engine
-    response = offers_engine.create_engine()
-    if not _check_errors(response):
-        return
-    print("Offer engine has been created.")
-
-    response = offers_engine.update_schema()
-    if not _check_errors(response):
-        return
-    print("Offer schema has been initialized.")
-
-    for response in offers_engine.update_synonyms():
-        if not _check_errors(response):
-            return
-    print("Offer synonyms have been set.")
+backend = appsearch.AppSearchBackend()
 
 
 def index_offers():
     # FIXME (dbaty, 2021-07-01): late import to avoid import loop of models.
     import pcapi.core.offers.models as offers_models
 
-    backend = appsearch.AppSearchBackend()
-
-    offers = offers_models.Offer.query.all()
-    documents = []
-    for offer in offers:
+    bookableOffers = []
```

Comment:
(stye) On utilise le snake_case dans notre code Python. (Pas facile quand on passe du code JavaScript au code Python ! :) )

```suggestion
    bookable_offers = []
```